tests/unittests/Foundation/NSBundleTests.mm

```diff
@@ -31,6 +31,13 @@
     ASSERT_EQ_MSG(nil, [NSBundle bundleWithPath:nil], @"Bundle with nil path somehow not nil");
 }
 
+// Add a new class that will definitely be part of the mainBundle for this UT executable.
+@interface Waffle : NSObject
+@property (nonatomic, assign) float bacon;
```

Comment:
lol

---
In reply to: [74169190](https://github.com/Microsoft/WinObjC/pull/740#discussion_r74169190) [](ancestors = 74169190)